test/common.bash

```diff
@@ -296,19 +299,25 @@ ch_cores_total=$((ch_nodes * ch_cores_node))
 ch_mpirun_np="-np ${ch_cores_node}"
 ch_unslurm=
 if [[ $SLURM_JOB_ID ]]; then
-    ch_multinode=yes     # can run on multiple nodes
-    ch_multiprocess=yes  # can run multiple processes
-    ch_mpirun_node='srun --ntasks-per-node 1'               # 1 rank/node
-    ch_mpirun_core="srun --ntasks-per-node $ch_cores_node"  # 1 rank/core
-    ch_mpirun_2='srun -n2'                                  # 2 ranks, 2 nodes
-    ch_mpirun_2_1node='srun -N1 -n2'                        # 2 ranks, 1 node
+    ch_mpirun_node='srun --ntasks-per-node 1'
+    ch_mpirun_core="srun --ntasks-per-node $ch_cores_node"
+    ch_mpirun_2_1node='srun -N1 -n2'
     # OpenMPI 3.1 pukes when guest-launched and Slurm environment variables
     # are present. Work around this by fooling OpenMPI into believing it's not
     # in a Slurm allocation.
     if [[ $ch_mpi = openmpi ]]; then
         # shellcheck disable=SC2034
         ch_unslurm='--unset-env=SLURM*'
     fi
+    if [[ $ch_nodes -eq 1 ]]; then
+        ch_multinode=
+        ch_multiprocess=
+        ch_mpirun_2=false
+    else
+        ch_multinode=yes
+        ch_multiprocess=yes
+        ch_mpirun_2='srun -n2'
+    fi
```

Comment:
The downside of setting `ch_multiprocess=` here is this means now a minimum of 2 nodes are required to run any of the host launch tests. Even though I would recommend folks use a guest launch in single node allocations I do think it is beneficial to test host launch. 

In general I feel if we detect Slurm env variables we should be able to run multiple processes. We could also still run `srun -n2`. If we want `srun -n2` to truly be only used for 2 ranks on 2 seperate nodes any tests using it would need to check `ch_multinode` and we may want to change it to `srun -N2 -n2` to be more explicit about what we are doing.

Reply:
I think `ch_multiprocess=` is a typo and I meant `ch_multiprocess=yes`.

Reply:
Can you make that fix as you test?